recipe/meta.yaml

```diff
@@ -1,15 +1,13 @@
 {% set name = "pyodbc" %}
-{% set version = "5.0.1" %}
+{% set version = "5.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 03d7d0b04d5a9156099ce8d03e92f3956783746fa9234eb6f5b5cfc12b645011
-  patches:         # [unix]
-    - setup.patch  # [unix]
```

Comment:
This patch was renamed in CF but wasn't removed, see https://github.com/conda-forge/pyodbc-feedstock/blob/main/recipe/meta.yaml#L11-L12. We should look at if we really need to remove it or if we need it at all.

Reply:
I wonder if that patch is still relevant. conda-forge added it 7 years ago https://github.com/conda-forge/pyodbc-feedstock/commit/df9f4976c4f4b288978c623bd8f46ba6199b2e75#diff-9b9492537c426331956c57a68b605459bf6b7a17b5401ef4a4c0a18f74b51a73R10. 
Yeah, it would be nice to verify that.

Reply:
It seems that adding sys.prefix/include to include_dirs has historically been relevant for ensuring Python headers were correctly included during builds, particularly in environments with custom Python installations. However, I removed it as recent builds have been successful without the patch, which suggests it may no longer be necessary in our current setup. Please let me know if you believe this should be re-added based on any specific requirements, or if you'd prefer we continue without it for now.

Reply:
pyodbc's files are the following for v5.0.1:
```
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/INSTALLER
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/LICENSE.txt
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/METADATA
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/RECORD
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/REQUESTED
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/WHEEL
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/direct_url.json
lib/python3.9/site-packages/pyodbc-5.0.1.dist-info/top_level.txt
lib/python3.9/site-packages/pyodbc.cpython-39-x86_64-linux-gnu.so
lib/python3.9/site-packages/pyodbc.pyi
```
see https://conda-metadata-app.streamlit.app/?q=pkgs%2Fmain%2Flinux-64%2Fpyodbc-5.0.1-py39h6a678d5_0.conda#raw-json
The same is true for the latest version 5.1.0 (from the logs):
```
  adding 'pyodbc.cpython-311-x86_64-linux-gnu.so'
  adding 'pyodbc.pyi'
  adding 'pyodbc-5.1.0.dist-info/LICENSE.txt'
  adding 'pyodbc-5.1.0.dist-info/METADATA'
  adding 'pyodbc-5.1.0.dist-info/WHEEL'
  adding 'pyodbc-5.1.0.dist-info/top_level.txt'
  adding 'pyodbc-5.1.0.dist-info/RECORD'
```
I think it's fine to drop the patch.